### PR TITLE
[microsoft_dhcp] Update Readme

### PR DESCRIPTION
**Pull request by cherryleaf**
Type of change

- Enhancement

## What does this PR do?

Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/microsoft_dhcp/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,14 @@
 # newer versions go on top
+- version: "1.3.2"
+  changes:
+    - description: Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing
```
Comment (bmorelli25) [inline packages/microsoft_dhcp/changelog.yml:4]:
```suggestion
    - description: Tidy up of Markdown in Readme.md, including adding link to developer's documentation
```

**Review by andrewkroh — APPROVED** [review]

**Comment by andrewkroh** [comment]
@cherryleaf Can you please resolve the merge conflicts. I would do it myself, but it appears you have not given permission for me to push (e.g. "Allow edits from maintainers.") (see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).

**Comment by andrewkroh** [comment]
Please reopen if you can "Allow edits from maintainers." or address the merge conflicts.

**Pull request closed without merging**